packages/access-api/wrangler.toml

```diff
@@ -25,6 +25,10 @@ binding = "__D1_BETA__"
 database_name = "spaces-dev"
 database_id = "7c676e0c-b9e7-4711-97c8-7b1c8eb229ae"
 
+[[r2_buckets]]
+binding = "ACCESS_API_R2"
+bucket_name = "access-api-dev-1"
```

Comment:
Proposal:
```suggestion
binding = "DELEGATIONS"
bucket_name = "access-api-delegations-dev-0"
```

Reply:
sgtm! I will create buckets following similar pattern for non-`dev` envs

Reply:
https://github.com/web3-storage/w3protocol/pull/603